docs/_static/nuts-network.yaml

```diff
@@ -0,0 +1,131 @@
+openapi: "3.0.0"
+info:
+  title: Nuts network API spec
+  description: API specification for RPC services available at the nuts-network
+  version: 0.1.0
+  license:
+    name: GPLv3
+paths:
+  /api/document:
+    get:
```

Comment:
I'll add an issue for now. It's only there since I wanted the CLI to be able to query the document log for analytics for troubleshooting (and not an elaborate interface for querying). I think you want a more extensive query interface in the future (filtering, ranging for timestamps and/or hashes (or consistency hashes), etc).

Until we have a lot of data missing query parameters/pagination isn't going to break things.